otx/cli/tools/export.py

```diff
@@ -56,6 +56,11 @@ def get_args():
         action="store_true",
         help="This flag indicated if model is exported in half precision (FP16).",
     )
+    parser.add_argument(
+        "--model-type",
```

Comment:
How about `--export-type` ? I saw `export_type` variable is used in many code lines.